openmetadata-ui/src/main/resources/ui/src/styles/components/button.less

```diff
@@ -69,3 +69,9 @@ button {
 .ant-btn-primary {
   box-shadow: none;
 }
+
+.ant-btn.btn-outlined {
+  border: 1px solid @primary-color;
+  color: @primary-color;
+  background: transparent;
+}
```

Comment:
.btn-outlined is custom class? if not this will impact across app, is that expected?

Reply:
Yes it's custom for us